netmiko/help.md

```diff
@@ -108,6 +105,5 @@ This plugin does not contain any troubleshooting information.
 
 ## References
 
-* [Netmiko](https://github.com/ktbyers/netmiko)
```

Comment:
Why did we remove this? Given the name of the plugin

Reply:
Adding, thought it was the description URL.